rdmo/core/xml.py

```diff
@@ -45,17 +45,48 @@ def validate_root(root: Optional[xmlElement]) -> Tuple[bool, Optional[str]]:
 
 
 def validate_and_get_xml_version_from_root(root: xmlElement) -> Tuple[Optional[Version], list]:
-    unparsed_root_version = root.attrib.get('version') or LEGACY_RDMO_XML_VERSION
-    root_version, rdmo_version = parse(unparsed_root_version), parse(RDMO_INSTANCE_VERSION)
-    if root_version > rdmo_version:
-        logger.info('Import failed version validation (%s > %s)', root_version, rdmo_version)
+    rdmo_version = parse(__version__)
+
+    # Extract version attributes from the XML root
+    unparsed_required_version = root.attrib.get('required')  # New required version field
+    unparsed_root_version = root.attrib.get('version') or LEGACY_RDMO_XML_VERSION  # Fallback to legacy default
+
+    # Validate the 'required' attribute if it exists
+    if unparsed_required_version:
+        try:
+            required_version = parse(unparsed_required_version)
+        except ValueError:
+            logger.info('Import failed: Invalid "required" format in XML (%s)', unparsed_required_version)
+            errors = [_('The "required" attribute in this RDMO XML file is not a valid version.')]
+            return None, errors
+
+        if required_version > rdmo_version:
+            logger.info('Import failed: Required version (%s) > RDMO instance version (%s)', required_version,
+                        rdmo_version)
+            errors = [
+                _('This RDMO XML file requires a newer RDMO version to be imported.'),
+                f'Required version: {required_version}, Current version: {rdmo_version}.'
+            ]
+            return None, errors
+
+    # Fallback to validate the legacy 'version' field
+    try:
+        xml_version = parse(unparsed_root_version)
+    except ValueError:
+        logger.info('Import failed: Invalid "version" format in XML (%s)', unparsed_root_version)
+        errors = [_('The "version" attribute in this RDMO XML file is not a valid version.')]
+        return None, errors
+
+    # RDMO 1.x can not import XMLs from RDMO 2.x
+    if rdmo_version < parse('2.0.0') and xml_version >= parse('2.0.0'):
+        logger.info('Import failed: Backwards compatibility is not supported for RDMO versions < 2.0.0')
         errors = [
-            _('This RDMO XML file does not have a valid version number.'),
-            f'XML Version ({root_version}) is greater than RDMO instance version {rdmo_version}'
+            _('This RDMO XML file was created with a newer version of RDMO and cannot be imported.'),
+            _('Backwards compatibility is not supported for RDMO versions lower than 2.0.0.')
```

Comment:
I think "Backwards compatibility" is the wrong term here. It is more like forward compatibility. Actually the whole block makes no sense, since it will only be shipped with RDMO 2.3.0 which is > 2.0.0, right? (Did I get something wrong?)

Reply:
This is for the case when a RDMO 1.x tries to import an XML from RDMO 2.x. 
It is maybe an edge-.. Ohw now I see 🙃, this code will not even exist in the RDMO 1.x 🤣 

I'll remove this, thanks for noticing!